cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,32 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+
+    cy.visit('https://react-redux.realworld.io');
+
+    cy.get('h1')
+      .should('contain.text', 'conduit');
+
+    cy.get('a[href*="#register"]')
+      .click();
+
+    cy.get('h1')
+      .should('contain.text', 'Sign Up');
+
+    cy.get('[placeholder=Username]')
+      .type(user.username);
+
+    cy.get('[placeholder=Email]')
+      .type(user.email);
+
+    cy.get('[placeholder=Password]')
+      .type(user.password);
+
+    cy.get('.btn').click();
 
+    cy.get('a.nav-link')
+      .should('contain', user.username);
```

Comment:
It's a good practice to assert that the user was redirected to the correct page after the registration. You could check the URL with `cy.url().should('include', '/expectedPath')`.